workloads/network-perf/README.md

```diff
@@ -1,211 +1,69 @@
 # Network Scripts
 
 The purpose of the network scripts is to run uperf workload on the Openshift Cluster.
-There are 4 types network tests:
-1. pod to pod using Hostnetwork
-2. pod to pod using SDN
-3. pod to pod using Service
-4. pod to pod using Multus (NetworkAttachmentDefinition needs to be provided)
+There are 5 types network tests:
+
+1. pod to pod using SDN: `WORKLOAD=pod2pod ./run.sh`
+2. pod to pod using SDN and network policy: `WORKLOAD=pod2pod NETWORK_POLICY=true ./run.sh`
+3. pod to pod using Hostnetwork: `WORKLOAD=pod2pod HOSTNETWORK=true ./run.sh`
+4. pod to service: `WORKLOAD=pod2svc ./run.sh`
```

Comment:
There was also the service ip one with network policy ```run_serviceip_network_policy_test_fromgit.sh```

Reply:
Added!